odoo/addons/base/models/res_partner.py

```diff
@@ -132,7 +132,7 @@ class Partner(models.Model):
     _description = 'Contact'
     _inherit = ['format.address.mixin', 'avatar.mixin']
     _name = "res.partner"
-    _order = "display_name"
+    _order = "display_name, id"
```

Comment:
Why?

Reply:
That's a good reason, keep it like that.
But I suggest to make a separate commit for that issue.